spec.md

```diff
@@ -530,6 +536,53 @@ message PluginCapability {
       // returned by NodeGetInfo to ensure that a given volume is
       // accessible from a given node when scheduling workloads.
       ACCESSIBILITY_CONSTRAINTS = 2;
+
+      // EXPANDABLE_ONLINE indicates that the volume can be expanded
+      // when published to a node.
+      // When a Plugin implements this capability it MUST implement
+      // either EXPAND_VOLUME Controller capability or EXPAND_VOLUME
+      // Node capability or both.
+      // When a plugin has EXPANDABLE_ONLINE capability and also
+      // has EXPAND_VOLUME controller capability then the plugin
+      // MUST support expansion of controller published volumes via
+      // ControllerExpandVolume.
+      // A plugin MAY support this capability if it does not have
+      // controller EXPAND_VOLUME capability but only node EXPAND_VOLUME
+      // capability.
+      // Example 1:
+      //    Given a shared filesystem volume (ex. GlusterFs) -
+      //    the Plugin will set EXPANDABLE_ONLINE capability
+      //    and implement EXPAND_VOLUME Controller capability
+      //    but no EXPAND_VOLUME on the Node.
+      // Example 2:
+      //    Given a block storage volume type(ex. EBS) -
+      //    the Plugin will set EXPANDABLE_ONLINE capability
+      //    and implement EXPAND_VOLUME Controller capability
+      //    and EXPAND_VOLUME Node capability.
+      // Example 3:
+      //    Given a volume that is only expandable on node,
+      //    the Plugin will set EXPANDABLE_ONLINE capability
+      //    and implement EXPAND_VOLUME on the Node but no
+      //    EXPAND_VOLUME on Controller.
+      EXPANDABLE_ONLINE = 3;
```

Comment:
I understand that you followed the pattern of `ACCESSIBILITY_CONSTRAINTS`. I feel that's a tech debt (`Service` is meant to be ether `Controller Service` or `Node Service`). I'd suggest the following
```protobuf
message PluginCapability {
  message Service {
    ...
  }

  message VolumeExpansion {
    enum Type {
      UNKNOWN = 0;
      ONLINE = 1;
      OFFLINE = 2;
    }
    Type type = 1;
  }

  oneof type {
    Service service = 1;
    VolumeExpansion volume_expansion = 2;
  }
}
```